src/generateApplicationFormJson.ts

```diff
@@ -93,6 +94,5 @@ axios(`${apiUrl}/baseFields`, {
     jsonOutput.close();
   });
 }).catch((error: AxiosError) => {
-  // eslint-disable-next-line no-console
-  console.log(error.response?.data);
+  logger.error({ error }, 'Error getting base fields');
```

Comment:
Also happy to change this if you want, @bickelj - this will definitely be a longer error message than there was before, and it that's not useful to you we can do something smaller.

Reply:
It works fine.

I would have expected the `{ error }` or `error` to follow the error message, though, as part of varargs rather than the first arg, but after trying the alternative and [looking at the docs](https://getpino.io/#/docs/api?id=mergingobject) this looks great.

I would rather have a more verbose log that I have to scroll through than have a terse log that is missing what I am looking for. So I have no objection to the longer message.

Reply:
Yeah, it's an API design choice pino made that I'm not really a fan of, but :shrug: .